### PR TITLE
prepare resolver 0.8.0 release

### PR DESCRIPTION
**Pull request by bluejekyll**
FYI @cssivision 

Once this passes tests, etc, I'll publish to crates.io

### CODE REVIEW

**Comment by cssivision** [comment]
great

File: proto/src/rr/domain/name.rs [hunk 1]
```diff
@@ -434,23 +435,27 @@ impl Name {
...
+    ///
+    /// // from_str calls through to from_utf8
+    /// let utf8_name = Name::from_str("WWW.example.COM.").unwrap();
+    /// let lower_name = Name::from_str("www.example.com.").unwrap();
     ///
     /// assert!(!bytes_name.eq_case(&utf8_name));
     /// assert!(lower_name.eq_case(&utf8_name));
     /// ```
-    pub fn from_utf8<S: AsRef<str>>(name: S) -> ProtoResult<Self> {
+    pub(crate) fn from_utf8<S: AsRef<str>>(name: S) -> ProtoResult<Self> {
```
Comment (cssivision) [inline proto/src/rr/domain/name.rs]: method `from_utf8` is private, but used in compatibility-tests

Comment (bluejekyll) [reply proto/src/rr/domain/name.rs]: yeah, I just noticed that I missed those... I'm anticipating needing to change this name during my mDNS work, which is why I privatized it.

**Comment by bluejekyll** [comment]
ok, I've fixed the issue locally. I'm content that it's good... publishing and merging.

**Comment by bluejekyll** [comment]
ok, published.

**Pull request merged**